### DISCUSSION TITLE
AUTH_AUTHORITY=<auth_authority_url>

### DISCUSSION THREAD

**Discussion by mileo** [shopinvader/shopinvader-template-nuxtjs#25] [Q&A]
Hi,

We are planning to contribute to this repo in the next months. 

Today we success deploy the app locally and didn't know how to configure the AUTH_AUTHORITY.

![image](https://user-images.githubusercontent.com/1975978/216446623-5866f8fa-46f0-49cb-90ed-224c71d8134a.png)

Probably a lack of JWT configuration in Odoo.

Thanks in advance

cc: @Todomir

**Comment by rbravo86** [comment]
Hi, 
The Auth part is not fully developed yet. 
I'll send to your e-mail a demo .env file which should let you keep working on the project.
